docs/source/lesson/prg/python_intro.rst

```diff
@@ -319,63 +205,65 @@ This interactive prompt does the following:
 This is why you may see the interactive loop referred to as a
 **REPL**: **R**\ead-**E**\valuate-**P**\rint-**L**\oop.
 
-Statements and Strings
-=================================================================
+Python 3 Features
+--------------------
+
+As mentioned earlier, we assume you will use Python 2.7.X because
+there are still some libraries that haven't been ported to
+Python 3. However, there are some features of Python 3 we can and want
+to use in Python 2.7. Before we do anything else, we need to make
+these features available to any subsequent code we write::
 
-Let us explore the syntax of Python.
-Type into the interactive loop and press Enter::
+  >>> from __future__ import print_function, division
```

Comment:
I think you need a short description about why these features are desirable to use. A single short sentence for each should be fine so as not to clutter. Otherwise it seems as if they are introduced with incomplete motivation